Packs/Salesforce/Integrations/Salesforce_IAM/Salesforce_IAM.py

```diff
@@ -0,0 +1,443 @@
+
+                iam_user_profile.set_result(success=True,
+                                            iden=user_id,
+                                            active=True,
+                                            action=IAMActions.UPDATE_USER,
+                                            details=res
+                                            )
+
+        return iam_user_profile
+
+    except Exception as e:
+        message, code = handle_exception(e)
+        iam_user_profile.set_result(success=False,
+                                    error_message=message,
+                                    error_code=code,
+                                    action=IAMActions.UPDATE_USER
+                                    )
+        return iam_user_profile
+
+
+def disable_user_command(client, args, mapper_out, is_command_enabled):
+    try:
+        user_profile = args.get("user-profile")
+        iam_user_profile = IAMUserProfile(user_profile=user_profile)
+
+        if not is_command_enabled:
+            user_profile.set_result(action=IAMActions.DISABLE_USER,
+                                    skip=True,
+                                    skip_reason='Command is disabled.')
+
+        else:
+            email = iam_user_profile.get_attribute('email')
+            user_id, _ = client.get_user_id_and_activity_by_mail(email)
+
+            if not user_id:
+                error_code, error_message = IAMErrors.USER_DOES_NOT_EXIST
+                iam_user_profile.set_result(action=IAMActions.DISABLE_USER,
+                                            error_code=error_code,
+                                            skip=True,
+                                            skip_reason=error_message)
+            else:
+                salesforce_user = iam_user_profile.map_object(mapper_name=mapper_out)
+                salesforce_user['IsActive'] = False
+                salesforce_user = {key: value for key, value in salesforce_user.items() if value is not None}
+                res = client.update_user_profile(user_term=user_id, data=salesforce_user)
+
+                iam_user_profile.set_result(success=True,
+                                            iden=user_id,
+                                            active=False,
+                                            action=IAMActions.DISABLE_USER,
+                                            details=res
+                                            )
+
+        return iam_user_profile
+
+    except Exception as e:
+        message, code = handle_exception(e)
+        iam_user_profile.set_result(success=False,
+                                    error_message=message,
+                                    error_code=code,
+                                    action=IAMActions.DISABLE_USER
+                                    )
+        return iam_user_profile
+
+
+def check_and_set_manndatory_fields(salesforce_user):
+    params = demisto.params()
```

Comment:
We should avoid global functions/variable. demisto.params() is global. Instead either pass these params to the functions from the main or add them to the Client